From e60f872c2dc4c1d9227977c8714373fe6328699c Mon Sep 17 00:00:00 2001
From: Chukun Pan <amadeus@jmu.edu.cn>
Date: Mon, 10 Feb 2025 15:01:21 +0800
Subject: [PATCH] arm64: dts: qcom: ipq6018: rename labels of mp5496 regulator

Change the labels of mp5496 regulator from ipq6018 to mp5496.

Suggested-by: Konrad Dybcio <konrad.dybcio@oss.qualcomm.com>
Suggested-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
Signed-off-by: Chukun Pan <amadeus@jmu.edu.cn>
Link: https://lore.kernel.org/r/20250210070122.208842-5-amadeus@jmu.edu.cn
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
---
 arch/arm64/boot/dts/qcom/ipq6018-mp5496.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/arch/arm64/boot/dts/qcom/ipq6018-mp5496.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq6018-mp5496.dtsi
@@ -7,26 +7,26 @@
 #include "ipq6018.dtsi"
 
 &cpu0 {
-	cpu-supply = <&ipq6018_s2>;
+	cpu-supply = <&mp5496_s2>;
 };
 
 &cpu1 {
-	cpu-supply = <&ipq6018_s2>;
+	cpu-supply = <&mp5496_s2>;
 };
 
 &cpu2 {
-	cpu-supply = <&ipq6018_s2>;
+	cpu-supply = <&mp5496_s2>;
 };
 
 &cpu3 {
-	cpu-supply = <&ipq6018_s2>;
+	cpu-supply = <&mp5496_s2>;
 };
 
 &rpm_requests {
 	regulators {
 		compatible = "qcom,rpm-mp5496-regulators";
 
-		ipq6018_s2: s2 {
+		mp5496_s2: s2 {
 			regulator-min-microvolt = <725000>;
 			regulator-max-microvolt = <1062500>;
 			regulator-always-on;
